src/components/ChangePlan/Checkout/Step/Step.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
How about adding some tests for this component?

Reply:
yes, sure. I will add some test

Reply:
Done